website/templates/landing.mako

```diff
@@ -79,7 +83,7 @@
                           </div>
                           <!-- /ko -->
                           <div>
-                              <button type="submit" class="btn btn-warning" data-bind="visible: !submitted()" id="signupSubmit">Sign up free</button>
+                              <button type="submit" class="btn btn-warning" data-bind="visible: !submitted(), css: {disabled: !password.isValid()}" id="signupSubmit">Sign up free</button>
```

Comment:
At least on some pages (this one), this causes the button to not look clickable until both:

1. the password is valid
2. the user has clicked somewhere since the password became valid

<img width="594" alt="screen shot 2016-05-31 at 16 04 12" src="https://cloud.githubusercontent.com/assets/5659262/15688897/69173898-274a-11e6-9824-3826d0770f48.png">

Not sure if this is/should be a blocker, but it seems like it might be confusing to the user.